scripts/remove-worktree.sh

```
@@ -0,0 +1,319 @@
+    blocked=true
+    return 0
+  fi
+
+  # What the remotes actually have, fresh — remote-tracking refs alone miss
+  # tags entirely. Fail closed only when NO remote answers: guessing while
+  # offline is how work gets lost.
+  local failed_remotes=""
+  for remote in $remotes; do
+    if remote_refs=$("$@" ls-remote --heads --tags "$remote" 2>/dev/null); then
+      reached=true
+      while read -r sha ref; do
```

Shellcheck (0.11.0)

[warning] ref appears unused. Verify use (or export if used externally).

(SC2034)